src/libraries/System.Text.Json/Common/JsonKnownNamingPolicy.cs

```diff
@@ -4,7 +4,7 @@
 namespace System.Text.Json.Serialization
 {
     /// <summary>
-    /// The <see cref="Json.JsonNamingPolicy"/> to be used at run-time.
+    /// The <see cref="Json.JsonNamingPolicy"/> to be used at run time.
```

Comment:
Why not just "runtime"?

Reply:
I am copying style that I have seen from @gewarren . runtime = CLR, run time = when you're running.